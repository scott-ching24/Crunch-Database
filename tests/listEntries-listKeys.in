set a 1 2
set b 2 3
set b 3 4
set c 4 5 
list keys
list entries
bye
