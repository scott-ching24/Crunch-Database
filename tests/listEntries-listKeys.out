> ok

> ok

> ok

> ok

> c
b
a

> c [4 5]
b [3 4]
a [1 2]

> bye
